api.py

```diff
@@ -333,6 +337,13 @@ class find_at_DGGS_cell(Resource):
     async def get(self, request, *args, **kwargs):
         """Calls DGGS table to query loci features by DGGS cell ID"""
         dggs_cell = str(next(iter(request.args.getlist('dggs_cell'))))
-        result = await find_at_dggs_cell(dggs_cell)
-        print(result)
-        return json(result, status=200)
+        p = re.compile('^[RSrs][0-9]+$')
```

Comment:
this is good. suggest we further limit the number to 10 numbers and the letter at the start to any alphabet, e.g. 
```
p = re.compile('^[a-zA-Z][0-9]{10}$')
```
This will enable the API to match any DGGS Level 10 Cell ID

Reply:
actually, according to github.com/GeoscienceAustralia/AusPIX_DGGS/blob/master/auspixdggs/auspixengine/dggs.py#L149, only `['N','O','P','Q','R','S']` are valid.

so this is probably more accurate:
```
p = re.compile('^[N-S][0-9]{10}$')
```

Reply:
Thanks,